opensrp-giz-malawi/src/test/java/org/smartregister/giz/adapter/VillageDoseAdapterTest.java

```diff
@@ -0,0 +1,64 @@
+package org.smartregister.giz.adapter;
+
+import android.content.Context;
+import android.view.View;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.robolectric.util.ReflectionHelpers;
+import org.smartregister.giz.domain.VillageDose;
+import org.smartregister.giz.viewholder.VillageDoseViewHolder;
+import org.smartregister.view.ListContract;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class VillageDoseAdapterTest {
```

Comment:
Update **all** tests to follow the convention `test[MethodName][ExpectedResult][Parameters] (Parameters optional if not present)`

Reply:
Done